pipeline-runner/R/qc-4-filter_gene_umi_outlier.R

```diff
@@ -82,17 +97,27 @@ filter_gene_umi_outlier <- function(scdata, config, sample_id, cells_id, task_na
   # get evenly spaced predictions on downsampled data for plotting lines
   xrange <- range(downsampled_data$log_molecules)
   newdata <- data.frame(log_molecules = seq(xrange[1], xrange[2], length.out = 10))
-  line_preds <- suppressWarnings(predict(fit, newdata, interval = "prediction", level = 1 - p.level))
 
-  line_preds <- cbind(newdata, line_preds) %>%
-    dplyr::select(-fit) %>%
-    dplyr::rename(lower_cutoff = lwr, upper_cutoff = upr)
+  pred_int_values <- sort(c(seq(0,1,0.01), 0.999, 0.9999, 0.99999, 0.999999))
```

Comment:
I don't think a p.level of 1 makes sense. That's a p.value of 0. So the confidence intervals would go to infinity.

Reply:
```suggestion
  pred_int_values <- sort(c(seq(0, 0.99, 0.01), 0.999, 0.9999, 0.99999, 0.999999))
```